portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -35,10 +38,25 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //get comments in the form of JSON
+    // Load comments from Datastore
+    Query query = new Query("Comment");
+    DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Let's make it a class variable like you did with `comment_list` so that we don't initialize this for every HTTP request